made changes to make it look more like the design

JMaager opened the pull request:
### What does this PR do?
I have added a few things and removed a few lines from the old code.

select-user.html:
i have added the nav bar on the left, not linked it to anything, purely styling. i have fixed some padding and margin to make the spacing between everything as close to pixel perfect as i can see. as it stands now, the styling for the header section does not seem to be complete. so have to look at it in the future most likely. 

admin-add-user.html:
I have also gone back and fixed the header section, side nav bar and spacing for the Create user page, so it fits more with the Manage User styling.

### Description of Task to be completed?
Build out UI components for listing users, editing roles, and adding new users.
Ensure forms are accessible and responsive.

### How should this be manually tested?
comparing with design

### Any background context you want to provide?
as we talked about in DM's i'd love if you can see how we can position the edit and delete button to fit more with the design, as the code is now i am kind of lost.
i think for bigger displays, there should be a space between the table data and edit and delete. so that the buttons to modify data is seperate from the data itself on big displays.
if what i said was confusing hope this image shows what i have in mind:
![image](https://github.com/user-attachments/assets/05d00766-cf07-4cee-84e3-821dd5d87760)

JMaager committed:
made changes to make it look more like the design

I have added a few things and removed a few lines from the old code.

i have added the nav bar on the left, not linked it to anything, purely styling.
i have fixed some padding and margin to make the spacing between everything as close to pixel perfect as i can see.
as it stands now, the styling for the header section does not seem to be complete. so have to look at it in the future most likely.
i have also gone back and fixed the header section, side nav bar and spacing for the Create user page, so it fits more with the Manage User styling.

kevindjenhansen reviewed (changes requested):
For the edit/delete button I figured out you could try changing it like this and it should align the buttons on the far right side:
`          <td class="p-4" style="text-align: right;">
            <a href="#"> <!-- link to manage student -->
              <button class="!text-primary">Edit</button>
            </a>
          </td>
          <td class="hidden p-4 sm:table-cell" style="text-align: right;"> <!--Need to style the checkbox:checked to show the modal overlay-->
            <input type="checkbox" id="delete-checkbox" aria-label="delete user" class="hidden">
            <button class="alert-button text-[16px]">Delete</button>
          </td>  `

There are a few other changes I'd like for the select-user.html

- Increase the font size/boldness for the "ID/First Name/Last Name/Year/Role/DOB" part of the table
- Increase spacing below between the row of buttons and back to dashboard button

There are a few other changes I'd like for the admin-add-user.html

- Increase the spacing below between the "Insert user information below" and the form
- Make the height of the input forms slightly bigger
- Increase the size of the buttons to fit more in line with the Figma design (look at the manage user page on Figma)

It would also be great if you applied the same to the manage-user.html as you do on the admin-add-user.html

JMaager committed:
Merge branch 'develop' into admin-select-user-page

JMaager committed:
Applied changes

Applied changes asked for in last comment.
also did some other styling to fit the design more. please look over.

kevindjenhansen reviewed (approved):
Great work with the implemented changes, it looks wonderful!

I'm gonna approve the work done so far, but I would like some changes with the mobile view in the future:
- When scrolling down the drop box menu seems to follow, I would like for that to be static on the top so that it doesn't follow the user when scrolling
- On select-user.html in order to match the Figma mobile design the "Add user" button should be hidden on mobile
- On the admin-add-user.html I would like for the buttons to on top of each other in a row and placed in the center (see image, or look at Figma mobile design for manage user)
![E6tthiK](https://github.com/user-attachments/assets/325cc5a2-c5cb-4b00-aced-825c8739364b)
